px-data-grid-renderer-mixin.html

```diff
@@ -69,18 +74,31 @@
       this._editing = false;
     }
 
+    _keyupListener(event) {
+      if (event.keyCode == 27) {
```

Comment:
`keyCode` is deprecated, use `key`. Take a look at https://github.com/vaadin/vaadin-overlay/pull/39 and keep in mind that for IE11 `e.key` will be `Esc` instead of standard `Escape`

Reply:
Done.